.circleci/config.yml

```diff
@@ -120,13 +120,13 @@ workflows:
           file: mapping/staging/mapping.txt
           app: $APP_CENTER_APP_NAME
           token: $APP_CENTER_TOKEN
-          version-name: $(grep "VERSION_NAME" testapp/build/generated/source/buildConfig/staging/com/rakuten/tech/mobile/miniapp/testapp/BuildConfig.java | awk '{print $7}' | tr -d \''";\')
+          version-name: $(grep "VERSION_NAME" ../testapp/build/generated/source/buildConfig/staging/com/rakuten/tech/mobile/miniapp/testapp/BuildConfig.java | awk '{print $7}' | tr -d '";')
```

Comment:
@khairul-alam-licon  Is this working? I think it shouldn't be using `../` because circleci runs the scripts from the root of the code directory. 

Also, I don't think that all of the build artifacts are being persisted to the workspace: only the APKs and the mappings currently are persisted (see https://github.com/rakutentech/android-buildconfig/blob/master/circleci/android-sdk/config.yml#L78).

So I think you need to add an additional step to `post-steps:` in the build job above so that `BuildConfig.java` is persisted to the workspace. Alternatively, it looks like the whole `build` directory for `miniapp` is being persisted to the workspace, so maybe you could just use the BuildConfig from that instead?

Reply:
Yeah, let me to check. Sorry, I haven't returned to this PR for long time 😅

Reply:
@corycaywood Yeah, I have updated the PR with persistence to the workplace. It's not showing the error anymore but looks like we need to merge this PR to see the effect.